### PR TITLE
Finalized VQE notebooks with EQSN backend and Scheduler 

### PR DESCRIPTION
**Pull request by Shiro-Raven**

### CODE REVIEW

File: examples/vqe-examples/general_scheduler.py [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+        ----------
+        num_likelihoods : int
+        hardware_config : HardwareConfig
+        allow_distributed : bool
+            True if distributed computing is allowed, False otherwise.
+            Distributed computing requires CCNOT gates for entanglement,
+            Qiskit does not currently (easily) support it.
+        """
+        self.num_likelihoods = num_likelihoods
+        self._oracle_size = oracle_size
```
Comment (stephendiadamo) [inline examples/vqe-examples/general_scheduler.py:42]: The naming convention from Andrea's version was specifically for QFT free QAE.. maybe we change `num_likelihoods` -> `num _circuits` and `_oracle_size` to `_circuit_width` to be more general.

File: examples/vqe-examples/general_scheduler.py [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+        
+        self.__make_schedule(couldNotFit, round+1)
+    
+    def __fill_allocation(self, possible_qpus: List[List[int]]) -> List[int]:
+        """
+        Allocate qubits
+        Parameters
+        ----------
+        possible_qpus : List of two-elements list [bits, index]
+        Returns
```
Comment (stephendiadamo) [inline examples/vqe-examples/general_scheduler.py:134]: A newline here between the end of params and return title makes the docstring slightly easier to read

File: interlinq/utils/vqe_subroutines.py [hunk 3]
```diff
@@ -11,23 +13,24 @@ def string_to_qutip_pauli(string):
...
+            obs_tensor_prod[index] = string_to_qutip_pauli(pauli)
+
+        obs_tensor_prod = tensor(obs_tensor_prod)
         
-        expectation = expect(needed_paulis, needed_matrices)
+        total += coefficient * np.vdot(vector, obs_tensor_prod @ vector) 
         
-        total += coefficient * expectation[0][0]
-    
     return total
```
Comment (stephendiadamo) [inline interlinq/utils/vqe_subroutines.py:36]: A new line at the end of the file is missing.

**Review by stephendiadamo — CHANGES_REQUESTED** [review]
Minor things to change, I'll check over the notebooks, but just the changes here should be made and I'll merge them in.

File: interlinq/utils/vqe_subroutines.py (hunk 3 above)
Comment (Shiro-Raven) [reply interlinq/utils/vqe_subroutines.py]: done

File: examples/vqe-examples/general_scheduler.py (hunk 2 above)
Comment (Shiro-Raven) [reply examples/vqe-examples/general_scheduler.py:135]: done

File: examples/vqe-examples/general_scheduler.py (hunk 1 above)
Comment (Shiro-Raven) [reply examples/vqe-examples/general_scheduler.py]: done

Comment (stephendiadamo) [reply examples/vqe-examples/general_scheduler.py]: Were the changes committed? I still see `num_likelihoods` and `_oracle_size`.

**Review by stephendiadamo — APPROVED** [review]

**Pull request merged**